### PR TITLE
Add Japanese translated README

### PR DESCRIPTION
**Pull request by eltociear**
# What does this PR do?

Add Japanese README

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Comment by eltociear** [comment]
@sgugger Thanks!
Added a fix to the this file.

**Review by younesbelkada — APPROVED** [review]
初めまして @eltociear 👋 !
Thanks a lot for this great PR! 
The Japanese translation looks perfect to me, I am really looking forward to merging this!
I would advocate to add something similar to the Chinese translated `README`, at the beginning of this `README` file, let's add a dictionary for future contributors and for mapping the important keywords:
```
<!---
A useful guide for English-Chinese translation of Hugging Face documentation
- Add space around English words and numbers when they appear between Chinese characters. E.g., 共 100 多种语言; 使用 transformers 库。
- Use square quotes, e.g.,「引用」

Dictionary

Hugging Face: 抱抱脸
token: 词符（并用括号标注原英文）
tokenize: 词符化（并用括号标注原英文）
tokenizer: 词符化器（并用括号标注原英文）
transformer: transformer（不翻译）
pipeline: 流水线
API: API (不翻译）
inference: 推理
Trainer: 训练器。当作为类名出现时不翻译。
pretrained/pretrain: 预训练
finetune: 微调
community: 社区
example: 当特指仓库中 example 目录时翻译为「用例」
Python data structures (e.g., list, set, dict): 翻译为列表，集合，词典，并用括号标注原英文
NLP/Natural Language Processing: 以 NLP 出现时不翻译，以 Natural Language Processing 出现时翻译为自然语言处理
checkpoint: 检查点
-->
```
Gently tagging our japanese student ambassadors @haruki-N @blackdoor571 @hiro819 if they want to have a quick look ;)

**Comment by sgugger** [comment]
Thanks! Can you also just quickly run `make fix-copies` to make the CI happy?

**Comment by sgugger** [comment]
@eltociear It looks like the start prompt you added is not present in the Japanese README. Could you make sure it's correct?

File: README_ja.md [hunk 1]
```diff
@@ -0,0 +1,482 @@
...
+
+> **_注意:_**  Windowsでは、キャッシュの恩恵を受けるために、デベロッパーモードを有効にするよう促されることがあります。このような場合は、[このissue](https://github.com/huggingface/huggingface_hub/issues/1062)でお知らせください。
+
+## モデルアーキテクチャ
+
+🤗Transformersが提供する **[全モデルチェックポイント](https://huggingface.co/models)** は、[ユーザー](https://huggingface.co/users)や[組織](https://huggingface.co/organizations)によって直接アップロードされるhuggingface.co [model hub](https://huggingface.co)からシームレスに統合されています。
+
+現在のチェックポイント数: ![](https://img.shields.io/endpoint?url=https://huggingface.co/api/shields/models&color=brightgreen)
+
+🤗Transformersは現在、以下のアーキテクチャを提供しています（それぞれのハイレベルな要約は[こちら](https://huggingface.co/docs/transformers/model_summary)を参照してください）:
```
Comment (eltociear) [inline README_ja.md:297]: @sgugger Thank you for pointing this out. start prompt is here.

File: utils/check_copies.py [hunk 2]
```diff
@@ -78,6 +78,14 @@
             " {paper_authors}.{supplements}"
         ),
     },
+    "README_ja.md": {
+        "start_prompt": "🤗Transformersは現在、以下のアーキテクチャを提供しています",
+        "end_prompt": "1. 新しいモデルを提供したいですか？",
```
Comment (sgugger) [inline utils/check_copies.py:83]:
```suggestion
        "end_prompt": "1. 新しいモデルを投稿したいですか？",
```
Sorry it's the end prompt the util cannot find. Looks like the line is indeed different in the Japanese README.

Comment (eltociear) [reply utils/check_copies.py:83]: @sgugger Thanks! There is no problem with the contents of that.

**Comment by sgugger** [comment]
Thanks again for your contribution!

**Comment by eltociear** [comment]
@sgugger THANKS too!

**Pull request merged**